src/apps/material/helper.ts

```diff
@@ -48,52 +54,70 @@ export const getWorkDescriptionListData = ({
   const allLanguages = mainLanguages
     .map((language) => language.display)
     .join(", ");
-  const fallBackManifestation = getWorkManifestation(work);
+  const fallBackManifestation = getWorkManifestation(
+    work,
+    "bestRepresentation"
```

Comment:
Please look at comment above about having a global entity describing the manifestation type